CONTRIBUTING.md

```diff
@@ -29,18 +29,73 @@ Guidelines](https://opensource.google.com/conduct/).
 
 ## Building and Testing the Connector
 
-The connector is build using sbt version 0.13. The following targets are used:
-* `sbt publishM2` - builds the connector and publishes it to the local maven repository
-* `sbt test` - runs the unit tests
-* `sbt it:test` - runs the integration tests. Those tests run various cases using a GCP project used for testing.
-* `sbt acceptance:test` - runs the acceptance test. Those test create several Dataproc clusters and then run several
-  test scripts using PySpark. Please run `sbt publishM2` before in order to build the tested artifacts.
+The connector is built using the Maven wrapper. The project contains several
+connectors, sharing some of the code among them. The connectors are:
+* spark-bigquery_2.11 - a Scala 2.11 based connector, targeting Spark 2.3 and
+  2.4 using Scala 2.11.
+* spark-bigquery_2.12 - a Scala 2.12 based connector, targeting Spark 2.4 and 
+  3.x using Scala 2.12.
+* spark-bigquery:spark2.4 - a Java only connector, targeting Spark 2.4 (of all
+  Scala versions), using the new DataSource APIs.
+* spark-bigquery:spark3 - a Java only connector, targeting Spark 3 (of all
+  Scala versions), using the new DataSource APIs. Still under development.
+
+The project's artifacts are:
+  
+* `spark-bigquery-parent` - The parent POM for all artifacts. Common settings
+  and artifact version should be defined here.
+* `bigquery-connector-common` - Utility classes for working with the BigQuery
+  APIs. This artifact has no dependency on Spark.
+* `spark-bigquery-connector-common`- Common code shared among all connectors
+  (Scala and Java alike).
```

Comment:
`bigquery-connector-common`  v/s  `spark-bigquery-connector-common` looks a bit confusing. Maybe we should add a line or two.